pac4j-core/src/main/java/org/pac4j/core/profile/UserProfile.java

```diff
@@ -316,7 +312,10 @@ public String getLinkedId() {
     }
 
     public void setLinkedId(final String linkedId) {
-        CommonHelper.assertNotNull("linkedId", linkedId);
+        if (this.linkedId != null) {
```

Comment:
If we want to remove a link to an account, we should allow to set `null`. We should just remove the `CommonHelper.assertNotNull` test.